### PR TITLE
Update UG from Draft UG

### PR DESCRIPTION
**Pull request by dloh2236**
Add UG for Contacts Management, test MarkBind formatting

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -74,29 +76,166 @@ Shows a message explaning how to access the help page.
...
+    * Note that each contact can have:
+        * At most one `[a/ADDRESS]`
+        * At most one `[c/COMPANY]` 
+        * At most one `[t/TELEGRAM_NAME]`
+</box>
+
+|   **Parameter**   | **Format**                                                                                                     | **Examples** (#g#Valid##/#r#Invalid##)                                                               |
+|:-----------------:|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+|      `NAME`       | Text up to 256 characters<br>Must be unique                                                                    | #g#Annie Dunkins##<br>#g#'Chewbaca' The 1st##                                                        |
+|  `PHONE_NUMBER`   | Numeric values<br>(optional "+" prefix)                                                                        | #g#81234567##<br>#g#+6581234567##<br>#r#A0u38niufd##<br>#r#(phone number cannot contain alphabets)## |
```
Comment (flexibo) [inline docs/UserGuide.md:161]: Maybe can make use of this instead

```diff
+ Annie Dunkins
+ 'Chewbaca' The 1st
+ 81234567
- A0u38niufd (phone number cannot contain alphabets)
```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -74,29 +76,166 @@ Shows a message explaning how to access the help page.
...
+
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [a/ADDRESS] [c/COMPANY] [t/TELEGRAM_NAME]`
+
+<box type="warning" seamless>
+    * `[a/ADDRESS]` should preferably be the company’s address
+    * Note that each contact can have:
+        * At most one `[a/ADDRESS]`
+        * At most one `[c/COMPANY]` 
+        * At most one `[t/TELEGRAM_NAME]`
+</box>
```
Comment (flexibo) [inline docs/UserGuide.md:156]: > [!NOTE]
> * [a/ADDRESS]` should preferably be the company’s address
> * Note that each contact can have:
>     * At most one `[a/ADDRESS]`
>     * At most one `[c/COMPANY]` 
>     * At most one `[t/TELEGRAM_NAME]`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -74,29 +76,166 @@ Shows a message explaning how to access the help page.
...
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+<box type="info" seamless>
+    * Using partial keywords will be matched. e.g. 'ha' will match 'hans'
+    * The search is case-insensitive. e.g `hAnS` will match `Hans`
+    * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
+    * Only the name is searched.
+    * Persons matching at least one keyword will be returned (i.e. `OR` search).
+      e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+</box>
```
Comment (flexibo) [inline docs/UserGuide.md:129]: > [!NOTE]
> * Using partial keywords will be matched. e.g. 'ha' will match 'hans'
> * The search is case-insensitive. e.g `hAnS` will match `Hans`
> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
> * Only the name is searched.
> * Persons matching at least one keyword will be returned (i.e. `OR` search).
>     * e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`

**Review by flexibo — CHANGES_REQUESTED** [review]

**Comment by nicholascher** [comment]
LGTM.

**Pull request merged**